data/json/speech.json

```
@@ -2526,6 +2526,24 @@
     "sound": "BARK!",
     "volume": 40
   },
+  {
+    "type": "speech",
+    "speaker": [ "mon_dog_samoyed", "mon_mi_go", "mon_mi_go_slaver", "mon_mi_go_myrmidon" ],
+    "sound": "BARK!",
+    "volume": 40
+  },
+  {
+    "type": "speech",
+    "speaker": [ "mon_dog_mutant_mongrel", "mon_mi_go", "mon_mi_go_slaver", "mon_mi_go_myrmidon" ],
+    "sound": "BARK!",
+    "volume": 40
+  },
+  {
+    "type": "speech",
+    "speaker": [ "mon_dog_mutant_loper", "mon_mi_go", "mon_mi_go_slaver", "mon_mi_go_myrmidon" ],
+    "sound": "BARK!",
+    "volume": 40
+  },
```

```suggestion
  {
    "type": "speech",
    "speaker": [ "mon_dog_samoyed", "mon_dog_mutant_mongrel", "mon_dog_mutant_loper", "mon_mi_go", "mon_mi_go_slaver", "mon_mi_go_myrmidon" ],
    "sound": "BARK!",
    "volume": 40
  },
```
Why not consolidate all of these into a single brick of code? That'd cut down on space and make things easier.